InvenTree/templates/js/translated/barcode.js

```diff
@@ -48,6 +52,44 @@ function makeBarcodeInput(placeholderText='', hintText='') {
     return html;
 }
 
+qrScanner = null;
+
+function startQrScanner() {
+    $('#barcode_scan_video_container').show();
+    qrScanner.start();
+}
+
+function stopQrScanner() {
+    if (qrScanner != null) qrScanner.stop();
+    $('#barcode_scan_video_container').hide();
+}
+
+function onBarcodeScanClicked(e) {
```

Comment:
@rkalman you need to export this function in the exported section - so just add 
`onBarcodeScanClicked`
in line 22 of this file